packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -1,19 +1,20 @@
 import { Configuration, Identity } from "@ory/client";
 import {
-  V0alpha2Api as OpenSourceV0alpha2Api,
-  AdminCreateIdentityBody,
+  CreateIdentityBody,
+  FrontendApi,
+  IdentityApi,
 } from "@ory/kratos-client";
 import { getRequiredEnv } from "../util";
 
 const publicUrl = getRequiredEnv("ORY_KRATOS_PUBLIC_URL");
 
-export const publicKratosSdk = new OpenSourceV0alpha2Api(
+export const publicKratosSdk = new FrontendApi(
   new Configuration({ basePath: publicUrl }),
 );
 
 const adminUrl = getRequiredEnv("ORY_KRATOS_ADMIN_URL");
 
-export const adminKratosSdk = new OpenSourceV0alpha2Api(
+export const adminKratosSdk = new IdentityApi(
```

Comment:
Maybe we can rename this accordingly to something like `kratosIdentityApi` to be more consistent with the changes made to kratos?